firebase-vscode/src/data-connect/emulator-stream.ts

```diff
@@ -59,19 +62,24 @@ export async function runEmulatorIssuesStream(
 /**
  * Based on the severity of the issue, either log, display notification, or display interactive popup to the user
  */
-export function displayAndHandleIssue(issue: EmulatorIssue, isPostgresEnabled: Signal<boolean>) {
+export function displayAndHandleIssue(
+  issue: EmulatorIssue,
+  isPostgresEnabled: Signal<boolean>,
+  schemaReloadFunc: () => void,
+) {
   const issueMessage = `Data Connect Emulator: ${issue.kind.toString()} - ${issue.message}`;
   if (issue.severity === Severity.ALERT) {
     vscode.window.showErrorMessage(issueMessage);
-  } else if (issue.severity === Severity.NOTICE) {
```

Comment:
Not blocking for this PR, but I wonder if we should chat to discuss if there should be a standard way for VSCode to handle different severity levels (i.e. would you prefer the `FILE_RELOAD` notification to be sent with `DEBUG` severity level, so you can keep displaying `NOTICE` as a warning message?)